server/src/metrics/actix_web_metrics.rs

```diff
@@ -158,28 +144,28 @@ pub(super) fn metrics_attributes_from_request(
         NETWORK_PROTOCOL_VERSION,
         http_version(req.version()),
     ));
-    attributes.push(SERVER_SOCKET_ADDRESS.string(conn_info.host().to_string()));
-    attributes.push(URL_PATH.string(http_target.to_owned()));
+    attributes.push(KeyValue::new(SERVER_ADDRESS, conn_info.host().to_string()));
+    attributes.push(KeyValue::new(URL_PATH, http_target.to_owned()));
     attributes.push(KeyValue::new(URL_SCHEME, http_scheme(conn_info.scheme())));
 
     let server_name = req.app_config().host();
     if server_name != conn_info.host() {
-        attributes.push(SERVER_ADDRESS.string(server_name.to_string()));
+        attributes.push(KeyValue::new(SERVER_ADDRESS, server_name.to_string()));
```

Comment:
Are we intentionally overriding SERVER_ADDRESS from line 147? I think we are using the server_name and only if not present we use conn_info.host(). If they're equal we could just use server_name and remove line 147... I think we can remove line 147 and remove the if in line 152

Reply:
Good point, this is copied from an example, so I guess I haven't really thought this through. Something to improve on when needed